keyboards/gurindam/readme.md

```diff
@@ -0,0 +1,23 @@
+# gurindam
+
+![Gurindam](https://cdn.discordapp.com/attachments/406032987243806721/922514068265369611/IMG_20211220_222843.jpg)
```

Comment:
Please reupload this on imgur.

Reply:
i've uploaded the image to the github. hope it's ok.